CHANGELOG.md

```diff
@@ -193,7 +193,7 @@ In order to use `vscode-languageclient` directly the compatibility layer was imp
   ```js
       resolve: {
           alias: {
-              'vscode': require.resolve('monaco-languageclient/lib/vscode-compatibility')
+              'vscode': require.resolve('monaco-languageclient/vscode-compatibility')
```

Comment:
Don't change an old CHANGELOG entry

Reply:
I think I replaced every occurence but you're right

btw that's the only reference to this, since the change is not needed using vite

Reply:
fixed, but I think we need to add something in the readme

Reply:
Yes, but LTGM